src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const totalRent = rentPerDay * days;
+
+  if (days >= 3 && days < 7) {
+    return totalRent - smallDiscount;
+  }
+
+  if (days >= 7) {
+    return totalRent - bigDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return totalRent - bigDiscount;
  } else if (days >= 3) {
    return totalRent - smallDiscount;
  }


```
You can swap these statements by place and get rid of one redundant check.
Also, use `if - else if` construction